docs/api/app.md

```diff
@@ -352,6 +352,16 @@ Returns:
 
 Emitted when the gpu process crashes or is killed.
 
+#### Event: 'renderer-process-crashed'
```

Comment:
This is one too many `#`, hence the bad `electron.d.ts` output that @alexeykuzmin noticed

Reply:
Fixed